src/app/main.js

```diff
@@ -1,10 +1,11 @@
-import { React, Component } from 'react';
-import { render } from 'react-dom';
-
+import React, { Component } from 'react'
+import { render } from 'react-dom'
+// import '../styles/main.css'
+import '../styles/main.sass'
```

Comment:
comment out this line and uncomment above line to check  if `.css` file is working or not